### PR TITLE
DX-1070 Set STDOUT.sync flag in Ruby

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: .docker/entrypoint/Gemfile.lock [hunk 1]
```diff
@@ -114,7 +118,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```
Comment (payex-dev) [inline .docker/entrypoint/Gemfile.lock:121]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5772335)

File: .docker/entrypoint/Gemfile.lock [hunk 2]
```diff
@@ -26,11 +26,13 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (asbjornu) [inline .docker/entrypoint/Gemfile.lock:29]: Shouldn't this have been removed in f7128fc?

File: .docker/entrypoint/Gemfile.lock [hunk 3]
```diff
@@ -26,11 +26,13 @@ GEM
...
     faraday (1.0.1)
       multipart-post (>= 1.2, < 3)
     fastimage (1.8.1)
       addressable (~> 2.3, >= 2.3.5)
     ffi (1.13.1)
+    ffi (1.13.1-x64-mingw32)
```
Comment (asbjornu) [inline .docker/entrypoint/Gemfile.lock:35]: Shouldn't this have been removed in f7128fc?

File: .docker/entrypoint/Gemfile.lock [hunk 4]
```diff
@@ -103,6 +105,8 @@ GEM
     multipart-post (2.1.1)
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
+    nokogiri (1.10.9-x64-mingw32)
```
Comment (asbjornu) [inline .docker/entrypoint/Gemfile.lock:108]: Shouldn't this have been removed in f7128fc?

File: .docker/entrypoint/Gemfile.lock [hunk 5]
```diff
@@ -156,6 +160,8 @@ GEM
       rb-inotify (~> 0.9, >= 0.9.7)
     sassc (2.4.0)
       ffi (~> 1.9)
+    sassc (2.4.0-x64-mingw32)
```
Comment (asbjornu) [inline .docker/entrypoint/Gemfile.lock:163]: Shouldn't this have been removed in f7128fc?

**Pull request merged**